From 376a09c8399670d1e5837825ee59269b5682f721 Mon Sep 17 00:00:00 2001
From: Florian Friesdorf <flo@chaoflow.net>
Date: Wed, 31 Aug 2011 22:14:53 +0200
Subject: [PATCH] doctest.NORMALIZE_WHITESPACE

---
 _pytest/doctest.py |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/_pytest/doctest.py b/_pytest/doctest.py
index aaebab7..a685778 100755
--- a/_pytest/doctest.py
+++ b/_pytest/doctest.py
@@ -72,7 +72,8 @@ class DoctestTextfile(DoctestItem, pytest.File):
         doctest = py.std.doctest
         failed, tot = doctest.testfile(
             str(self.fspath), module_relative=False,
-            optionflags=doctest.ELLIPSIS,
+            # XXX: make this configurable
+            optionflags=doctest.ELLIPSIS | doctest.NORMALIZE_WHITESPACE,
             raise_on_error=True, verbose=0)
 
 class DoctestModule(DoctestItem, pytest.File):
@@ -84,4 +85,4 @@ class DoctestModule(DoctestItem, pytest.File):
             module = self.fspath.pyimport()
         failed, tot = doctest.testmod(
             module, raise_on_error=True, verbose=0,
-            optionflags=doctest.ELLIPSIS)
+            optionflags=doctest.ELLIPSIS | doctest.NORMALIZE_WHITESPACE)
-- 
1.7.6

